src/main/java/org/jabref/gui/openoffice/OpenOfficePanel.java

```diff
@@ -229,7 +230,11 @@ private void initPanel() {
         pushEntriesAdvanced.setOnAction(e -> pushEntries(CitationType.AUTHORYEAR_INTEXT, true));
         pushEntriesAdvanced.setMaxWidth(Double.MAX_VALUE);
 
-        update.setTooltip(new Tooltip(Localization.lang("Ensure that the bibliography is up-to-date")));
+        if (currentStyle instanceof CitationStyle) {
+            update.setTooltip(new Tooltip(Localization.lang("Make bibliography")));
+        } else if (currentStyle instanceof JStyle) {
+            update.setTooltip(new Tooltip(Localization.lang("Make or update the bibliography")));
+        }
```

Comment:
this needs to be dependend on the observable. The currentStyle can switch between JStyle and CSL without the OpenOfficePanel reloading.

Reply:
No longer required as refresh feature implemented for CSL as well. Will have a unified tooltip now.